core/local/steps/add_infos.js

```diff
@@ -21,13 +21,14 @@ module.exports = function (buffer /*: Buffer */, opts /*: { syncPath: string } *
     for (const event of events) {
       try {
         if (event.action !== 'initial-scan-done') {
+          // TODO if (event.kind === 'symlink') emit an error
           event._id = id(event.path)
           if (['created', 'modified', 'renamed'].includes(event.action)) {
             log.debug({path: event.path, action: event.action}, 'stat')
-            event.stats = await fse.stat(path.join(opts.syncPath, event.path))
+            event.stats = await stater.stat(path.join(opts.syncPath, event.path))
           }
           if (event.stats) { // created, modified, renamed, scan
-            event.docType = event.stats.isDirectory() ? 'directory' : 'file'
+            event.docType = stater.kind(event.stats)
```

Comment:
Feels like mixing *doctypes* with *kinds*, although the latter can be more than just `file` or `directory`...
Should work now but may break at some point.